### PR TITLE
Feature sigup login oauth #12 and #13

### PR DESCRIPTION
**Pull request by jassalss**
This PR is for signup and login pages (Authentication google oauth). Almost all functionality is completed (FE and BE). I made many changes on the React side. Please run `npm install`  after I combined my branch with Dev. Please review my PR and share your feedback.

### CODE REVIEW

File: client/src/pages/LogIn.js [hunk 1]
```diff
@@ -16,6 +16,8 @@ const LogIn = () => {
       };
       localStorage.setItem("greetings", JSON.stringify(greetings));
       history.push("/auth-with-google");
+    } else {
+      alert("Rigth now, We only support gmail. Please use gmail.");
```
Comment (freddy2659) [inline client/src/pages/LogIn.js]: Typo: Right now

File: client/src/pages/SignUp.js [hunk 2]
```diff
@@ -16,6 +16,8 @@ const SignUp = () => {
       };
       localStorage.setItem("greetings", JSON.stringify(greetings));
       history.push("/auth-with-google");
+    } else {
+      alert("Rigth now, We only support gmail. Please use gmail.");
```
Comment (freddy2659) [inline client/src/pages/SignUp.js]: Typo: Right now

File: server/app.py [hunk 3]
```diff
@@ -1,10 +1,21 @@
 from flask import Flask
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
+from api.auth_handler import create_auth_blueprint
+from api.after_auth_handler import create_after_Auth_blueprint
+from flask_cors import CORS
+from config import basic_auth_configSetup
 
 app = Flask(__name__)
+app.secret_key = "Simerpreet"
```
Comment (freddy2659) [inline server/app.py]: What is the secret key used for? Is it something that could be in the .env/.example.env instead?

**Review by freddy2659 — CHANGES_REQUESTED** [review]
I think just the typos and maybe moving the secret to the .env is needed. Can you check out the conflicts with .gitignore and server/config.py as well and join them?

File: server/app.py (hunk 3 above)
Comment (jassalss) [reply server/app.py]: oh! sorry. Secret key is for session object. It should be random so no one can guess. I made changes. Its now random.

File: client/src/pages/SignUp.js (hunk 2 above)
Comment (jassalss) [reply client/src/pages/SignUp.js]: Thanks!  Fixed now

**Review by freddy2659 — APPROVED** [review]

**Review by freddy2659 — APPROVED** [review]

**Review by sina-jamshidi — COMMENTED** [review]
This looks great! Thanks!

**Pull request merged**